core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -263,8 +263,11 @@
       "The maximum size for each write-ahead log. See comment for property tserver.memory.maps.max"),
   TSERV_WALOG_MAX_AGE("tserver.walog.max.age", "24h", PropertyType.TIMEDURATION, "The maximum age for each write-ahead log."),
   TSERV_WALOG_TOLERATED_CREATION_FAILURES("tserver.walog.tolerated.creation.failures", "50", PropertyType.COUNT,
-      "The maximum number of failures tolerated when creating a new WAL file within the period specified by tserver.walog.failures.period."
-          + " Exceeding this number of failures in the period causes the TabletServer to exit."),
+      "The maximum number of failures tolerated when creating a new WAL file."
+          + " Exceeding this number of failures consecutively trying to create a new WAL causes the TabletServer to exit."),
+  TSERV_WALOG_TOLERATED_WRITING_FAILURES("tserver.walog.tolerated.writing.failures", "1000", PropertyType.COUNT,
```

Comment:
Should this default be higher?

Reply:
This would be a separate issue, but I think it would be more user friendly to make this a time based config.  For example retry for up to 30 min.  Also its nice to have the option to retry forever.  

If DFS is grumpy for a bit, do we really want all tservers to exit?  This would also be a separate issue, but it would be nice if the behavior was different when all tservers are having problems vs a few tservers are having problems.  Maybe this is best handled outside of Accumulo, with tservers providing enough info for external system to make decisions about killing tservers. Basically the health check concept.  An external system can query all tservers for health info and make decsions about killing individual ones.

Reply:
That does sound reasonable.  The existing monitor might already be able to serve this function but a separate process would work as well.
I am thinking for a bug release that I will remove the halting mechanism as that is really changing how things work.  In 2.0.0 however we might want that and indeed may want a better way to configure this such as using time like you suggested.  I will create a separate ticket for that.